certification/engine/engine.go

```diff
@@ -70,6 +77,22 @@ func queryChecks(checkName string) certification.Check {
 	return nil
 }
 
+var (
+	pyxisProjectId string
+	pyxisApiToken  string
+)
+
+func getPyxisParams() {
+	pyxisProjectId = viper.GetString("certification_project_id")
```

Comment:
This is unnecessary. I've added code to make sure that it is neither empty nor starts with a prefix. You can just call viper.GetString() where it is needed. This function can go away. See #492

Reply:
yup makes it cleaner 👍